### ISSUE TITLE
Postgrey integration (wish)

### ISSUE THREAD

**Issue by fff0x** [instrumentisto/docker-mailserver#11]
Hi, please consider to integrate a greylog solution; As for alpine linux the package [postgrey](https://pkgs.alpinelinux.org/packages?name=postgrey&branch=&repo=&arch=x86_68) is available. The configuration is minimal and grey logging could be disabled by default. Let me know if you need help with the configuration, if you plan to integrate it.

**Comment by tyranron** [comment]
@mbuelte, no problems. But better to do it after [first stable release](https://github.com/instrumentisto/docker-mailserver/issues/6).

**Comment by fff0x** [comment]
Sounds great!

**Comment by pedromorgan** [comment]
Yes please.. am about to migrate from gmail workspace back to self hosted..at home
